pkg/resources/k8s_services.go

```diff
@@ -36,9 +36,9 @@ func newAuthService(authorinoName, serviceNamespace string) *k8score.Service {
 func newMetricService(authorinoName, serviceNamespace string) *k8score.Service {
 	serviceName := "controller-metrics"
 	return newService(serviceName, serviceNamespace, authorinoName, k8score.ServicePort{
-		Name:       "https",
-		Port:       8443,
-		TargetPort: intstr.FromString("https"),
+		Name:       "http",
+		Port:       8080,
+		TargetPort: intstr.FromString("http"),
```

Comment:
Does the pod exports a port named "http"?

Reply:
Not really, there are no named ports exposed in the pod. I think we can remove this TargetPort and if needed in the future expose the metrics port via named port in the pod, wdyt?